frontend/src/views/Market.vue

```diff
@@ -478,10 +493,18 @@ export default Vue.extend({
 
       await Promise.all(nftIds.map(async nftId => {
         const price = (await this.lookupNftPrice(nftId))!;
-
         void price;
         this.nftPricesById[nftId] = price;
       }));
+
+    },
+
+    sortList( List:  Record<any, any>, direction: string): any {
```

Comment:
please don't use an uppercase variable name; it can be confused with an enum or constant